pluggy/hooks.py

```diff
@@ -293,6 +293,15 @@ def _maybe_apply_history(self, method):
                 res = self._hookexec(self, [method], kwargs)
                 if res and proc is not None:
                     proc(res[0])
+    
+    def clear_history(self):
+        """Clear the call history.  Only historic calls made after the most
+        recent call to this method will be replayed onto newly-registered
+        plugins.  Helpful for reusing plugin managers between, for example,
```

Comment:
This *Helpful for reusing plugin managers between...* is far to specific to be in this doc-string.